### PR TITLE
get class in doPriv

### LINKED ISSUE #14041: DoPriv needed in ResourceLoaderImpl
We have a call to getClassloader without wrapping it in a doPriv block

### PR DESCRIPTION
**Pull request by benjamin-confino**
fixes #14041

### CODE REVIEW

File: dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/ResourceLoaderImpl.java [hunk 1]
```diff
@@ -55,7 +57,6 @@ public void cleanup()
      * @see org.jboss.weld.resources.spi.ResourceLoader#classForName(java.lang.String)
      */
     @Override
-    @FFDCIgnore(ClassNotFoundException.class)
```
Comment (tevans78) [inline dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/ResourceLoaderImpl.java:58]: I think you might still need this one

Comment (benjamin-confino) [reply dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/ResourceLoaderImpl.java]: And its back

**Comment by benjamin-confino** [comment]
#build

**Review by tevans78 — APPROVED** [review]

**Comment by benjamin-confino** [comment]
#run-libby-bot

**Pull request merged**